server/app/assets/javascripts/preview.ts

```diff
@@ -261,9 +233,7 @@ class PreviewController {
       questionType &&
       questionType.textContent === PreviewController.STATIC_QUESTION_TEXT
     if (useAdvancedFormatting) {
-      const contentElement = PreviewController.formatText(text, true)
-      contentElement.classList.add('text-sm')
-      contentElement.classList.add('font-normal')
```

Comment:
Removing these classes to be in line with what will actually render. They were removed from the backend in this PR: https://github.com/civiform/civiform/pull/5495/files#diff-02a0a04dcd13a5a42384a58108ed56ad7ded9a847d5f0084e2285f7134dd92ca